src/mapport.cpp

```diff
@@ -268,7 +176,7 @@ static void DispatchMapPort()
 
```

Comment:
Above here:
```cpp
    if (g_mapport_enabled_protos & g_mapport_current_proto) {
        // Enabling another protocol does not cause switching from the currently used one.
        return;
    }
```
This code is either dead, or the comment is outdated, as we can no-longer "enable another protocol".

Reply:
It was both. Removed outdated comments and dead code in the latest push. I cleaned up this function in the follow-up.